scaaml/stats/print_stats.py

```diff
@@ -145,6 +161,29 @@ def print(self) -> None:
                 for j, trace_stat in enumerate(
                         self._split_trace_stats[split][trace_name]):
                     # Check that we are filling the right row.
-                    assert trace_stat.__class__.__name__ == table[j + 1][0]
+                    assert class_name(trace_stat) == table[j + 1][0]
                     table[j + 1][i + 1] = trace_stat.result()
         print(tabulate(table, headers='firstrow'))
+
+        print('')
+        print('-------------------------------------------')
+        print('')
+
+        print('Trace statistics per group:')
+        for trace_name in self._measurements_info:
```

Comment:
As this function is becoming a bit long and it naturally has "sections",  it might be good to split it.

_print_trace_statistics()
_print_trace_statistics_per_group()
etc

Up to you if you want to split it (I'm fine either way)